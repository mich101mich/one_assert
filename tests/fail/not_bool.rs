fn main() {
    one_assert::assert!(5);
    one_assert::assert!(5, "hi");
    one_assert::assert!(1 + 2);
    one_assert::assert!(1 + 2, "hi");
}
