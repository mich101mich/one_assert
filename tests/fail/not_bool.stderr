error[E0308]: mismatched types
 --> tests/fail/not_bool.rs:2:25
  |
2 |     one_assert::assert!(5);
  |                         ^ expected `bool`, found integer

error[E0308]: mismatched types
 --> tests/fail/not_bool.rs:3:25
  |
3 |     one_assert::assert!(5, "hi");
  |                         ^ expected `bool`, found integer

error[E0308]: mismatched types
 --> tests/fail/not_bool.rs:4:25
  |
4 |     one_assert::assert!(1 + 2);
  |                         ^^^^^ expected `bool`, found integer

error[E0308]: mismatched types
 --> tests/fail/not_bool.rs:5:25
  |
5 |     one_assert::assert!(1 + 2, "hi");
  |                         ^^^^^ expected `bool`, found integer
